api/services/Frontend.js

```diff
@@ -6,7 +6,7 @@ throughout the code.
 
 */
 
-const prefix = `${process.env.PROTOCOL}://${process.env.DOMAIN}`
+const prefix = `${process.env.PROTOCOL}://${process.env.FRONTEND_DOMAIN}`
```

Comment:
Ah. For some reason the other places where I saw it in use lead me to
believe that it was to represent the domain at which the server/api was
operating.

On Aug 4, 2016 6:47 PM, "Lawrence Wang" <notifications@github.com> wrote:

> In api/services/Frontend.js
> <https://github.com/Hylozoic/hylo-node/pull/157#discussion_r73616126>:
>
> > @@ -6,7 +6,7 @@ throughout the code.
> >
> >  */
> >
> > -const prefix = `${process.env.PROTOCOL}://${process.env.DOMAIN}`
> > +const prefix = `${process.env.PROTOCOL}://${process.env.FRONTEND_DOMAIN}`
>
> why did you find this to be necessary? DOMAIN is supposed to be the
> frontend domain already
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/157/files/9af6e8a7c9f6ef38172173a00cbf64a2a97311ff#r73616126>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABWAYY47GqFSiukP1OsDGKXmle9HmfOgks5qcmv_gaJpZM4Jc836>
> .
>